components/talksPage/talksPage.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import style from './talksPage.module.scss';
+import data from '../../assets/videos.json';
+import YouTube from 'react-youtube';
+
+function TalksPage() {
+  const talksList = data;
+  const [currSrc, setSrc] = useState('GHbNaDSWUX8');
+
+  const calcArrowSide = (curr, side, length) => {
+    if (side > curr) {
+      if (curr === length) {
+        return 1;
+      }
+    } else if (side < curr) {
+      if (curr === 1) {
+        return length;
+      }
+    }
+    return side;
+  };
+
+  const onVideoClick = (event) => {
+    event.target.pauseVideo(); // stop chosen video
```

Comment:
why is the pausing needed?

Reply:
Cause when you choose a video in the carousels, you want the video to play on the big iframe. So I stop the small video from playing at the same time.